packages/app/src/client/viewsBasic/tenantIntegrations/tabs/Installed.tsx

```diff
@@ -47,6 +48,7 @@ const useStyles = makeStyles(theme => ({
 
 export const InstalledIntegrations = withIntegrationListFromParams(
   ({ integrationList }: IntegrationListProps) => {
+    // This integrationList needs to be mapped to integrationDefs (at least to access the Status component, and possibly Edit button)
```

Comment:
@terrcin

Reply:
@MatApple a nice way to do it would be that whenever integrations are loaded their respective definition is added to the object so you could easily go `integration.def...`.

At the moment you can import `integrationDefRecords` from `viewsBasic/integrationDefs/index.ts` and get it with `integrationDefRecords[integration.kind]`

Reply:
I can do that as part of updating the k8s-logs layout/stying

Reply:
Awesome sounds good!